### PR TITLE
readthedocs does not provide access to /docs/examples/*; fixes #1578

### LINKED ISSUE #1578: readthedocs does not provide access to /docs/examples/*
In /docs/examples/config, /docs/examples/deployment, /docs/examples/password-reset there are several example files with names not ending in .rst.

There are some references to these files within https://moin-20.readthedocs.io but afaik there is no way to view these files. 

Wiki admins installing moinmoin from pypi have no means of viewing these files because the docs directory is not included in the distribution.

Preferred solution is for readthedocs to have working links to the files.

### PR DESCRIPTION
**Pull request by RogerHaase**
some of the files moved to contrib/* are not referenced in the docs

moin.fcgi, moin.wsgi, and test.wsgi were moved but not changed, black reformatted after push

### CODE REVIEW

File: docs/admin/backup.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ Selective Backup
 ================
 If you want a backup of MoinMoin and your data, then backup the following:
 
-* your data
+* your data, ususlly everything under wiki/
```
Comment (ThomasWaldmann) [inline docs/admin/backup.rst:19]: typo

File: docs/admin/configure.rst [hunk 2]
```diff
@@ -564,7 +564,8 @@ username (like with german umlauts or accented characters). If moin does not
 crash (log a Unicode Error), you have likely found the correct coding.
 
 For users configuring GivenAuth on Apache, an example virtual host configuration
-file is included with MoinMoin in `docs/examples/deployment/moin-http-basic-auth.conf`.
+is included at `/contrib/deployment/moin-http-basic-auth.conf`
```
Comment (ThomasWaldmann) [inline docs/admin/configure.rst:567]: no leading "/"?

File: docs/admin/configure.rst [hunk 3]
```diff
@@ -1570,13 +1571,13 @@ https://docs.python.org/3/library/logging.config.html#configuration-file-format
 
 
 There are also some logging configurations in the
-`docs/examples/config/logging/` directory.
+`contrib/logging/` directory.
```
Comment (ThomasWaldmann) [inline docs/admin/configure.rst:1574]: better here!

File: docs/admin/configure.rst (hunk 2 above)
Comment (ThomasWaldmann) [reply docs/admin/configure.rst:567]: (also applies at some other places)

File: docs/admin/configure.rst (hunk 3 above)
Comment (RogerHaase) [reply docs/admin/configure.rst:1574]: Thanks for corrections, will fix

**Pull request merged**